### ISSUE TITLE
"org.webjars.npm" % "core-js"not included in web-modules

### ISSUE THREAD

**Issue by roti** [sbt/sbt-web#173]
[core-js](https://www.npmjs.com/package/core-js) is not copied to `target/web/web-modules/main/webjars/lib` during the assets phase. Every other npm webjars I tried is, but for some reason `core-js` is not. I can provide a test project if needed. Steps to reproduce would be:

* create new play project
* add following dependencies
```
libraryDependencies += "org.webjars.npm" % "core-js" % "3.2.0"
libraryDependencies += "org.webjars.npm" % "decimal.js" % "10.2.0"
```
* run `sbt assets`

Result: `decimal.js` is present in `target/web/web-modules/main/webjars/lib`, `core-js` is not.

**Comment by spencerhakim** [comment]
The jars produced by WebJars over the last 2 weeks (Aug 1st-14th) don't contain any directory entries, and sbt-web seems to fail to extract them as a result. It's discussed in [this ticket](https://github.com/webjars/webjars/issues/1863). This issue is therefore not limited to just `core-js`, but to however many artifacts were published in those 2 weeks.

**Comment by mkurz** [comment]
Closing. Either this is fixed by now or, according to #1863, needs a new webjars-locator version in sbt-web. Upcoming sbt-web 1.5.0 depends on latest webjars-locator. I already released 1.5.0-M1 for testing purposes. So at latest with the 1.5 release this should be fixed IMHO.
